__tests__/app.test.js

```diff
@@ -144,4 +143,4 @@ describe("GET /api/articles/:article_id/comments", () => {
             expect(response.body.msg).toBe('bad request')
         })
     })
```

Comment:
I think you'd pass this test if you had it but you're missing a test to check you get a 400 if you try and get the comments of an article with an invalid id
like `/api/articles/katherine/comments`